firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -428,10 +436,9 @@ private void sendSessionLog(String name, Timer startTime, Timer endTime) {
   /**
    * Only send screen trace if FrameMetricsAggregator exists.
    *
-   * @param activity The Activity for which we're monitoring the screen rendering performance.
    * @return true if supported, false if not.
    */
-  private boolean isScreenTraceSupported(Activity activity) {
+  protected boolean isScreenTraceSupported() {
```

Comment:
Can FrameMetricsAggregator not exist? It seems that we are forcing to create one as a part of the initializer and so, curious how this helps.

Reply:
I think this is checking if the Class itself exists, because the class itself was only added in later Android versions. I only removed the parameter because it is not used at all, I assume it was used before for some other code that got removed.

Reply:
Sounds good.